packages/storybook-ui/src/modules/ui/components/assets/svg_package.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable */
+(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
Thank you @danielduan 
It's a legacy solution from PR for Storybook 2.0 :)
I think we can switch to the "normal" way to import svg.
We'll need to change some loaders in webpack config, for that. Actually, I think we need to do it very carefully!